shared/js/intl_helper.js

```diff
@@ -0,0 +1,170 @@
+(function(global) {
+  'use strict';
+
+  /*************************
+
+  IntlHelper.define('shortDate', 'datetime', {
+    year: 'numeric',
+    month: 'long',
+  });
+
+  function setValue() {
+    var formatter = IntlHelper.get('shortDate');
+    console.log(formatter.format(new Date()));
+  }
+
+  setValue();
+
+  IntlHelper.observe('shortDate', setValue);
+  
+   ****************************/
+  var objectCache;
+
+  const knownObjects = {
+    'datetime': {
+      set: function(options) {
+        const customOptions = Object.assign({}, options);
```

Comment:
Are you sure you want to use `const` in this helper?  Will this require `;version=1.8` when linking to it?